settings/motorExtensions.cmd

```diff
@@ -2,6 +2,6 @@
 # Create a soft motor record on top of a normal motor record which allow the gap to be set for the motor
 # Copy this file to C:\Instrument\Settings\config\<machine>\configurations\galil
 
-$(IFNOTTESTDEVSIM) < $(GALILCONFIG)/xyBeamstop.cmd
-$(IFTESTDEVSIM) < $(MOTOREXTENSIONS)/settings/xyBeamstop/xyBeamstop.cmd
+$(IFNOTTESTDEVSIM) < $(GALILCONFIG)/settings/xyBeamstop/xyBeamstop.cmd
```

Comment:
I still don't agree with this directory structure. It implies the `xyBeamstop.cmd` should be in the directory:

```
C:\Instrument\Settings\config\NDW1695\configurations\galil\settings\xyBeamstop
```

That is at odds with where other configuration files are stored. Everything else is stored in the top level galil folder. I think this should be in the original location `$(GALILCONFIG)/xyBeamstop.cmd` so that it is consistent with other motor extensions. Please just make sure that `C:\Instrument\Apps\EPICS\support\motorExtensions\master\settings\README` is updated to indicate where the settings file should be copied to.